### PR TITLE
add Azure.AI.Vision.ImageAnalysis 0.9.0-beta.1

### PR DESCRIPTION
**Pull request by BrianMouncer**
We released public packages last Friday, and the public announcement is tomorrow.

### CODE REVIEW

**Review by czubair — APPROVED** [review]

**Comment by BrianMouncer** [comment]
#sign-off

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -378,8 +378,8 @@
 "Microsoft.Azure.Management.VideoAnalyzer","","1.0.0-beta.3","Resource Management - Video Analyzer","Video Analyzer","NA","NA","NA","","false","","","","","","","","","","",""
 "Microsoft.Azure.ServiceBus.EventProcessorHost","5.0.1","","Service Bus Event Processor Host","Service Bus","NA","NA","NA","","false","","","","active","","","","","","",""
 "Azure.IoT.TimeSeriesInsights","","1.0.0-beta.1","Time Series Insights","IoT","NA","NA","NA","","false","","","","","","","","","","",""
-"Azure.AI.Vision.Core","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"Azure.AI.Vision.Core","","0.9.0-beta.1","Core","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","","false","","","","","","","Azure.AI.Vision.Core","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:381]: Maybe this should be something like `Vision Core` ... I wouldn't expect it to be a Core library that the other libraries use: 

image

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -378,8 +378,8 @@
...
+"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","ImageAnalysis","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","","false","","","","","","","Azure.AI.Vision.ImageAnalysis","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:382]: Following the rest of our package names I think we should probably use `Image Analysis` with a space. 

Also the repo location isn't particularly useful here if the repo is not open source.

Comment (danieljurek) [reply _data/releases/latest/dotnet-packages.csv:382]: I also don't think we should have this package replacing itself in the `Replaces` field. That should be empty.

File: _data/releases/latest/java-packages.csv [hunk 3]
```diff
@@ -713,7 +713,7 @@
 "project","com.microsoft.azure.sessionmanager","","1.0.0-beta.1","Session Management - Project","Session Manager","NA","NA","NA","client","false","","","","","","","","","","",""
 "spark-streaming-eventhubs_2.10","com.microsoft.azure","1.6.3","","Spark - Streaming Eventhubs","Apache Spark Connector","NA","NA","NA","client","false","","","","","","true","","","","",""
 "spark-streaming-eventhubs_connector_2.10","com.microsoft.azure","1.6.3","","Spark - Streaming Eventhubs Connector","Apache Spark Connector","NA","NA","NA","client","false","","","","","","true","","","","",""
-"client-sdk","com.microsoft.cognitiveservices.speech","1.24.2","","Speech","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","client","false","","","","","","","","","","",""
+"client-sdk","com.microsoft.cognitiveservices.speech","1.26.0","","Speech","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","client","false","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/java-packages.csv:716]: Let's make this change in another PR unless this package is part of the release.

File: _data/releases/latest/python-packages.csv [hunk 4]
```diff
@@ -361,7 +361,7 @@
 "mltable","1.2.0","","Machine Learning - Table","Machine Learning","NA","NA","NA","","false","","","","","","","","","","",""
 "azure-purview-administration","","1.0.0b1","Purview Administration","Purview","purview","NA","NA","","false","","","","","","","","","","",""
 "text-analytics","1.0.2","","Text Analytics","Cognitive Services","NA","NA","NA","","false","","","","","","","","","","",""
-"azure-ai-vision","","0.9.0b1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"azure-ai-vision","","0.9.0b1","Azure AI Vision SDK","Cognitive Services","NA","NA","NA","","false","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/python-packages.csv:364]: LGTM

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (danieljurek) [reply _data/releases/latest/dotnet-packages.csv:381]:
```suggestion
"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","","NA","NA","","false","","","","","","","","","","",""
```

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (danieljurek) [reply _data/releases/latest/dotnet-packages.csv:382]:
```suggestion
"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","","NA","NA","","false","","","","","","","","","","",""
```

File: _data/releases/latest/java-packages.csv (hunk 3 above)
Comment (danieljurek) [reply _data/releases/latest/java-packages.csv:716]:
```suggestion
"client-sdk","com.microsoft.cognitiveservices.speech","1.24.2","","Speech","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","client","false","","","","","","","","","","",""
```

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (czubair) [reply _data/releases/latest/dotnet-packages.csv:381]: @achandmsft can you help with the naming here.

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (czubair) [reply _data/releases/latest/dotnet-packages.csv:382]: > Following the rest of our package names I think we should probably use `Image Analysis` with a space.
> 
> Also the repo location isn't particularly useful here if the repo is not open source.

Looking at this again - agree with Daniel on both counts. I am not sure about the naming, though, so added Arun to help out.

File: _data/releases/latest/dotnet-packages.csv [hunk 5]
```diff
@@ -378,8 +378,8 @@
 "Microsoft.Azure.Management.VideoAnalyzer","","1.0.0-beta.3","Resource Management - Video Analyzer","Video Analyzer","NA","NA","NA","","false","","","","","","","","","","",""
 "Microsoft.Azure.ServiceBus.EventProcessorHost","5.0.1","","Service Bus Event Processor Host","Service Bus","NA","NA","NA","","false","","","","active","","","","","","",""
 "Azure.IoT.TimeSeriesInsights","","1.0.0-beta.1","Time Series Insights","IoT","NA","NA","NA","","false","","","","","","","","","","",""
-"Azure.AI.Vision.Core","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:381]:
```suggestion
"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","NA","NA","NA","","true","","","","","","","","","","",""
```

File: _data/releases/latest/dotnet-packages.csv [hunk 6]
```diff
@@ -378,8 +378,8 @@
...
+"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:382]:
```suggestion
"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","NA","NA","NA","","true","","","","","","","","","","",""
```

**Review by danieljurek — APPROVED** [review]
Approving to get this into docs build

File: _data/releases/latest/dotnet-packages.csv [hunk 7]
```diff
@@ -378,8 +378,8 @@
 "Microsoft.Azure.Management.VideoAnalyzer","","1.0.0-beta.3","Resource Management - Video Analyzer","Video Analyzer","NA","NA","NA","","false","","","","","","","","","","",""
 "Microsoft.Azure.ServiceBus.EventProcessorHost","5.0.1","","Service Bus Event Processor Host","Service Bus","NA","NA","NA","","false","","","","active","","","","","","",""
 "Azure.IoT.TimeSeriesInsights","","1.0.0-beta.1","Time Series Insights","IoT","NA","NA","NA","","false","","","","","","","","","","",""
-"Azure.AI.Vision.Core","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","NA","NA","NA","","true","","","","","","","","","","",""
+"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","NA","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:382]:
```suggestion
"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","cognitiveservices","NA","NA","","true","","","","","","","","","","",""
```

File: _data/releases/latest/dotnet-packages.csv [hunk 8]
```diff
@@ -378,8 +378,8 @@
 "Microsoft.Azure.Management.VideoAnalyzer","","1.0.0-beta.3","Resource Management - Video Analyzer","Video Analyzer","NA","NA","NA","","false","","","","","","","","","","",""
 "Microsoft.Azure.ServiceBus.EventProcessorHost","5.0.1","","Service Bus Event Processor Host","Service Bus","NA","NA","NA","","false","","","","active","","","","","","",""
 "Azure.IoT.TimeSeriesInsights","","1.0.0-beta.1","Time Series Insights","IoT","NA","NA","NA","","false","","","","","","","","","","",""
-"Azure.AI.Vision.Core","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","NA","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:381]:
```suggestion
"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","cognitiveservices","NA","NA","","true","","","","","","","","","","",""
```

File: _data/releases/latest/dotnet-packages.csv [hunk 9]
```diff
@@ -378,8 +378,8 @@
 "Microsoft.Azure.Management.VideoAnalyzer","","1.0.0-beta.3","Resource Management - Video Analyzer","Video Analyzer","NA","NA","NA","","false","","","","","","","","","","",""
 "Microsoft.Azure.ServiceBus.EventProcessorHost","5.0.1","","Service Bus Event Processor Host","Service Bus","NA","NA","NA","","false","","","","active","","","","","","",""
 "Azure.IoT.TimeSeriesInsights","","1.0.0-beta.1","Time Series Insights","IoT","NA","NA","NA","","false","","","","","","","","","","",""
-"Azure.AI.Vision.Core","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
-"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","cognitiveservices","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:381]:
```suggestion
"Azure.AI.Vision.Core","","0.9.0-beta.1","Vision Core","Cognitive Services","<html>
<body>
<!--StartFragment-->

https://msasg.visualstudio.com/Skyman/_git/Carbon
--


<br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html>","NA","NA","","true","","","","","","","","","","",""
```

File: _data/releases/latest/dotnet-packages.csv [hunk 10]
```diff
@@ -378,8 +378,8 @@
...
+"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","cognitiveservices","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/dotnet-packages.csv:382]:
```suggestion
"Azure.AI.Vision.ImageAnalysis","","0.9.0-beta.1","Image Analysis","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","","true","","","","","","","","","","",""
```

File: _data/releases/latest/python-packages.csv [hunk 11]
```diff
@@ -361,7 +361,7 @@
 "mltable","1.2.0","","Machine Learning - Table","Machine Learning","NA","NA","NA","","false","","","","","","","","","","",""
 "azure-purview-administration","","1.0.0b1","Purview Administration","Purview","purview","NA","NA","","false","","","","","","","","","","",""
 "text-analytics","1.0.2","","Text Analytics","Cognitive Services","NA","NA","NA","","false","","","","","","","","","","",""
-"azure-ai-vision","","0.9.0b1","Unknown Display Name","Unknown Service","NA","NA","NA","","false","","","","","","","","","","","Needs Review"
+"azure-ai-vision","","0.9.0b1","Azure AI Vision SDK","Cognitive Services","cognitiveservices","NA","NA","","true","","","","","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/python-packages.csv:364]:
```suggestion
"azure-ai-vision","","0.9.0b1","Azure AI Vision SDK","Cognitive Services","https://msasg.visualstudio.com/Skyman/_git/Carbon","NA","NA","","true","","","","","","","","","","",""
```
